README.md

```diff
@@ -0,0 +1,17 @@
+## Online Examination System
+
+>> An online examination is the conducting of an examination or test online in order to measure students academic performance.
```

Comment:
An online examination is the conducting of an examination or test online in order to measure students academic performance